pkg/collector/corechecks/embed/apm.go

```diff
@@ -43,6 +43,10 @@ func (c *APMCheck) String() string {
 	return "APM Agent"
 }
 
+func (c *APMCheck) Version() string {
+	return ""
```

Comment:
This is a good question ! I hesitated with executing trace/jmx/process to get their version.

That would mean executing the binary twice (once for the version and once to run it) and I don't know if it would be useful. Their version should be sync with the agent and are easily accessible by the user.

In the end I didn't do it because there is no way for the user to update them (since we don't provide packages for them). So I don't think it's worth the effort.

What do you think ?

Reply:
OK, so since the versions are in-sync with the agent, I think that makes sense. 

With `JMXFetch` I think it would help as it's definitely not aligned. I think I have an idea for JMXFetch using the release.json and `-X` ldflag that would probably work nicely. But I'd rather do it in a separate PR (I can work on that).